README.md

```diff
@@ -202,34 +202,34 @@ Typically, you can resolve this error by indicating to your browser that you are
+The MediaCo sample application (available to Pega licensed users) includes an OAuth Client Registration record that it uses for authentication in your Infinity server (available in Security | OAuth 2.0 Client Registration): **MediaCoOauth** (for the Portal and Embedded use cases).
 
-You may use these records. If you want to create your own OAuth 2.0 Client Registration record, please refer to the **How to create OAuth2 registration in Infinity** section found below.
+You may use this existing registration record. If you want to create your own OAuth 2.0 Client Registration record, please refer to the **How to create OAuth2 registration in Infinity** section found below.
 
-* For the **Embedded** use case, you will use the OAuth 2.0 Client Registration record’s **Client ID** and **Client secret** as the values for **mashupClientId** and **mashupClientSecret** in the SDK’s **sdk-config.js** file.
+* For the **Embedded** use case, you will use the OAuth 2.0 Client Registration record’s **Client ID** as the value for **mashupClientId** in the SDK’s **sdk-config.js** file.
 
 * For the **Portal** use case, you will use the OAuth 2.0 Client Registration record’s **Client ID** as the value of **portalClientId** in the SDK’s **sdk-config.js** file.
 
 
 To ensure that the application is redirected to the proper page after authentication succeeds, you may need to update the OAuth 2.0 Client Registration record’s **List of redirect URIs** shown in the record’s **Supported grant types** section.
 
-The MediaCoOauth and MediaCoOauthNoLogin records that are included with the MediaCo sample application include the necessary redirect URIs for the default configuration:
+The MediaCoOauth record that is included with the MediaCo sample application include the necessary redirect URIs for the default configuration:
 
-* http://localhost:3502/auth.html and https://localhost:3502/auth.html for the Portal use case
+* http://localhost:3502/portal and https://localhost:3502/portal for the Portal use case
 
-*	http://localhost:3502/mashup/auth.html and https://localhost:3502/mashup/auth.html for the Embedded use case
+*	http://localhost:3502/embedded/auth.html and https://localhost:3502/embedded/auth.html for the Embedded use case
 
 If you configure your installation to have the React SDK static content served from a different **host:port** than the default, you should add new Redirect URIs to the list:
 
 * In the **Supported grant types** section add the following URLS to the list of redirect URLs by clicking on the + sign. (Note that the default port is 3502.)
 
-  * http://\<**host name or IP address of React SDK server**>:<**port you’re using**>/auth.html (for the portal use case)
+  * http://\<**host name or IP address of React SDK server**>:<**port you’re using**>/portal (for the portal use case)
```

Comment:
I would leave http: ... auth.html and https: ... /auth.html as the first two as they were but change to "(for the embedded use case as well as for any full re-authentication scenarios for any sample page)